src/Shared/TelemetryInitializers/DomainNameRoleInstanceTelemetryInitializer.cs

```diff
@@ -7,7 +7,6 @@
     using System.Threading;
```

Comment:
this is still in `namespace Microsoft.ApplicationInsights.AspNetCore.TelemetryInitializers` and also do we really want to make this public in WorkerService?